unit/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ list(REMOVE_ITEM sources
     ${CMAKE_CURRENT_SOURCE_DIR}/util/expr_cast/expr_undefined_casts.cpp
 )
 
-add_executable(unit ${sources} ${headers})
+add_executable(unit ${sources} ${headers} goto-programs/goto_trace_output.cpp)
```

Comment:
I think this does not want to go in as we glob all the cpp files in unit already (@reuk?) (CLion adds this automatically which is possibly something we should see if we can turn off...)

Reply:
I think this might be what is causing the failure on travis...